### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by Morten-Moeller**
Add profile page
Transfer button styling into utility.css
Embed SVG files into code to change the color on activation

### CODE REVIEW

File: src/bookmark.html [hunk 1]
```diff
@@ -36,7 +36,9 @@ <h2 class="quiz-card__heading">Quiz Card</h2>
           Lorem ipsum dolor sit, amet consectetur adipisicing elit. Est ducimus
           laboriosam
         </p>
-        <button class="quiz-card__button">Show answer here!</button>
+        <button class="quiz-card__button button-style" type="button">
```
Comment (anniviz) [inline src/bookmark.html]: why did you add the type attribute? i am not quite sure what it does but for my understanding it is not necessary for the state of the app right now.

File: src/bookmark.html [hunk 2]
```diff
@@ -74,7 +76,9 @@ <h2 class="quiz-card__heading">Quiz Card</h2>
           Lorem ipsum dolor sit, amet consectetur adipisicing elit. Est ducimus
           laboriosam voluptatem adipisci expedita unde ullam.
         </p>
-        <button class="quiz-card__button">Show answer here!</button>
+        <button class="quiz-card__button button-style" type="button">
```
Comment (anniviz) [inline src/bookmark.html]: same here

File: src/form.html [hunk 3]
```diff
@@ -12,40 +12,46 @@
       <h1 class="app-header__heading">Create quiz card</h1>
     </header>
 
-    <main class="mt-10 main-app">
+    <main class="main-app">
```
Comment (anniviz) [inline src/form.html:15]: i guess it would be a cleaner git workflow to open or reopen a branch for the form and not making so many changes in the profile branch ;o)

File: src/form.html [hunk 4]
```diff
@@ -12,40 +12,46 @@
...
+            id="input-tags"
             class="form__textarea"
             type="text"
-            id="input-tags"
             name="input-tags"
+            required
           />
+          <button
+            class="form__button button-style"
+            aria-label="Submit new quiz card"
```
Comment (anniviz) [inline src/form.html:50]: is there more than one submit button? if not it may not be needed to add the detailed aria-label

File: src/styles/_base.css [hunk 5]
```diff
@@ -12,7 +12,10 @@
   margin: 0;
   display: grid;
   grid-template-columns: 1fr min(40ch, calc(100% - 32px)) 1fr;
-  grid-template-rows: 60px auto 80px;
+  grid-template-rows: 48px auto 72px;
+  position: fixed;
+  top: 0;
+  left: 0;
```
Comment (anniviz) [inline src/styles/_base.css]: why do you need to do this?
also line 14, why does this have to be so complicated
just curious

File: src/styles/app-footer/app-footer.css [hunk 6]
```diff
@@ -1,5 +1,5 @@
 .app-footer {
-  background: var(--color-primary);
+  background-image: linear-gradient(170deg, #474c4e 0%, #2d3233 100%);
```
Comment (anniviz) [inline src/styles/app-footer/app-footer.css]: why not just background? should be working as well

**Review by anniviz — COMMENTED** [review]
nice work!

File: src/bookmark.html (hunk 1 above)
Comment (macfrei) [reply src/bookmark.html]: Right. A `button` element is already of `type="button"` so you don't need it here.

File: src/profile.html [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+  <body class="app-body">
+    <header class="app-header">
+      <h1 class="app-header__heading">Profile</h1>
+    </header>
+    <main class="main-app">
+      <section class="profile">
+        <img src="../../assets/anonymous.png" alt="" class="profile__image" />
+        <h2 class="profile__name">Generic</h2>
+        <p class="profile__about">
+          Hi i am an storybook element and i am very happy!
```
Comment (macfrei) [inline src/profile.html:19]: 🤣

File: src/profile.html [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+        </ul>
+        <button class="profile__button button-style" type="button">
+          Logout
+        </button>
+      </section>
+    </main>
+
+    <footer class="app-footer">
+      <nav class="app-footer__nav">
+        <a href="./index.html">
```
Comment (macfrei) [inline src/profile.html]: You might want to add aria-labels to the anchor elements so that the user knows where this link goes to.

File: src/styles/profile/profile.css [hunk 9]
```diff
@@ -0,0 +1,46 @@
+.profile {
+  padding: 16px;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  column-gap: 16px;
+  row-gap: 16px;
```
Comment (macfrei) [inline src/styles/profile/profile.css]: If you have two values which are the same, I'd recommend using the shorthand:
```suggestion
 grid-gap: 16px;
```

**Review by macfrei — COMMENTED** [review]
Well done 🚀

File: src/form.html (hunk 4 above)
Comment (Morten-Moeller) [reply src/form.html:50]: I added the aria label so the understanding is better if you cant see the button or page you are on. 
Of course its not necessary but i would prefer a more detailed button description.

File: src/styles/app-footer/app-footer.css (hunk 6 above)
Comment (Morten-Moeller) [reply src/styles/app-footer/app-footer.css]: Yesterday i tried it without and he didn't show me the gradient. But today it worked :D i don't know whats changed since yesterday gut you are right.

File: src/styles/_base.css (hunk 5 above)
Comment (Morten-Moeller) [reply src/styles/_base.css]: Line 14 is for wider screens so you have a minimum and a maximum that includes the column 1 and 3 so you have the possibility to use a bleeding edge effect for victory or something. With line 16 to 18 you are right, i tried to solve the problem with the grid gap at the end of the main element what is kinda hack and i am not sure if i really need this lines. But i change back to a div with an white-space utility class because its more clear what the code does.

File: src/profile.html (hunk 8 above)
Comment (Morten-Moeller) [reply src/profile.html]: good point thank you
